addon/components/o-s-s/input-container.hbs

```diff
@@ -1,7 +1,8 @@
 <div class="fx-col">
   <div
     class="oss-input-container
-      {{if this.hasStyleKey (concat ' oss-input-container--' this.styleKey)}}
+      {{if (or @errorMessage @hasError) ' oss-input-container--errored'}}
+      {{if @feedbackMessage (concat ' oss-input-container--' @feedbackMessage.type)}}
```

Comment:
Nit:
This could have probably been a bit clearer using a getter defined in the TS code instead